File: aliyun-python-best-practise/使用API管理RDS实例/describeDBinstances_demo.py
from aliyunsdkcore.client import AcsClient
from aliyunsdkrds.request.v20140815.DescribeDBInstancesRequest import DescribeDBInstancesRequest
client = AcsClient('access_key_id', 'access_key_secret', 'region_id')
request = DescribeDBInstancesRequest()
response = client.do_action_with_exception(request)
print(response)

File: aliyun-net-best-practice/DescribeDBInstancesDemo.cs
using System;
using System.Collections.Generic;
using Aliyun.Acs.Core;
using Aliyun.Acs.Core.Exceptions;
using Aliyun.Acs.Core.Profile;
using Aliyun.Acs.Rds.Model.V20140815;

namespace RdsDemo
{
    class DescribeDBInstancesDemo
    {
        public static void DescribeDBInstances()
        {
            IClientProfile profile = DefaultProfile.GetProfile("cn-hangzhou", "<accessKeyId>", "<accessSecret>");
            DefaultAcsClient client = new DefaultAcsClient(profile);

            var request = new DescribeDBInstanceAttributeRequest();
            request.DBInstanceId = "<dBInstanceId>";
            try {
                var response = client.GetAcsResponse(request);
                Console.WriteLine(System.Text.Encoding.Default.GetString(response.HttpResponse.Content));
            }
            catch (ServerException e)
            {
                Console.WriteLine(e);
            }
            catch (ClientException e)
            {
                Console.WriteLine(e);
            }
        }
    }
}